components/utilities/search.js

```diff
@@ -196,15 +204,23 @@ const Search = () => {
               opacity: 0,
               // left: '-40em'
             }}
-            className="algolia"
+            className={styles.ModalContainer}
           >
             <FocusTrap>
               <div
-                className="modalContainer"
+                className={styles.Modal}
+                style={{ background: "rgb(163, 168, 184, .6)" }}
```

Comment:
Why use inline styles? Can this be put in a class instead? Same below.

Also: why not use tailwind? For example: `bg-gray-40/60`

Reply:
Fixed on f0b5331